test-cypress/support/commands.js

```diff
@@ -0,0 +1,408 @@
+});
+
+let clickImportUrlBtn = Cypress.Commands.add('clickImportUrlBtn', () => {
+    cy.get('#wb-import-importUrl').click();
+});
+
+let verifyFileUpload = Cypress.Commands.add('verifyFileUpload', (importURL) => {
+    cy.get('#wb-import-fileInFiles').should('contain', importURL);
+});
+
+let verifyMessageOccurs = Cypress.Commands.add('verifyMessageOccurs', (message) => {
+    cy.get('#wb-import-fileInFiles').then(() => {
+        cy.get('.d-inline-block > :nth-child(2)').should('contain', message);
+    })
+});
+
+let selectRDFFormat = Cypress.Commands.add('selectRDFFormat', (rdfFormat) => {
+    cy.get('.modal-footer > :nth-child(2) > .btn-group > .btn').click({force: true});
+
+    cy.get('.modal-footer > :nth-child(2) > .btn-group > .dropdown-menu > li', {timeout: 1000})
+        .each(($btn, index) => {
+            if ($btn.text().trim() === rdfFormat) {
+                cy.get('.modal-footer')
+                // Need to be in modal footer context to use this
+                // method in all dialog windows
+                    .within(() => {
+                        cy.get(`:nth-child(${(index + 1)}) > .dropdown-item`).click({force: true});
+                    });
+            }
+        })
+});
+
+Cypress.Commands.add('clickImportButton', () => {
+    cy.get('[class="menu-item-icon icon-import"]')
+        .click({force: true});
+});
+
+Cypress.Commands.add('selectRDFLinkButton', () => {
+    cy.get('[href="import"]')
+        .click({force: true});
+});
+
+
+Cypress.Commands.add('selectRepo', (repoId) => {
+    cy.get('#repositorySelectDropdown')
+        .click({force: true})
+        .then(() => {
+            cy.get('.dropdown-item', {timeout: 1000})
+                .contains(repoId)
+                .click({force: true});
+        });
+});
+
+Cypress.Commands.add('selectRepoS', (repoId) => {
+    cy.get(".ot-loader-new-content").should("not.be.visible");
+    cy.get(".ot-loader").should("not.be.visible");
+    cy.get(".ot-main-loader").should("not.be.visible");
+    cy.get('#wb-repositories-repositoryInGetRepositories tr:contains("' + repoId + '") .icon-connection-off', {timeout: 2000})
+        .should("be.visible", {timeout: 1000});
+    cy.get('#wb-repositories-repositoryInGetRepositories tr:contains("' + repoId + '") .icon-connection-off', {timeout: 2000})
+        .click();
+});
+
+
+let clickImportBtnOnPopUpMenu = Cypress.Commands.add('clickImportBtnOnPopUpMenu', () => {
+    cy.get('.modal-footer > .btn-primary')
+        .click().wait(3000);
+});
+
+let removeUploadedFiles = Cypress.Commands.add('removeUploadedFiles', () => {
+    cy.get('#import-user > [files-table=""] > [ng-hide="loader"] > .mb-1 > ' +
+        '.form-inline.ng-pristine > .btn-link > .form-check-input')
+        .check({force: true}).should('be.checked');
+
+    cy.get('#wb-import-removeEntries')
+        .click({force: true});
+
+});
+
+let resetStatusOfUploadedFiles = Cypress.Commands.add('resetStatusOfUploadedFiles', () => {
```

Comment:
These selectors are horrible. Who needs to write such a selector when it has an id?!?